index.html

```diff
@@ -119,24 +116,33 @@ <h2 class="content-head content-head-ribbon">What do we do?</h2>
       <h2 class="content-head is-center">How do we do it?</h2>
 
       <p>
-        We have a <strong>3-step approach</strong> to getting your idea(s) tested
-        in the hands of <em>real</em> people.
+        We have a <strong>3-step &quot;Lean&quot; approach</strong>
+        to getting idea(s) tested in the hands of <em>real</em> people.
       </p>
       <ol>
         <li>
-          <i>Understand</i> the problem that needs to be solved
-          and confirm the intended users feel the <i>pain</i>
-          and <i>need</i> for a solution.
+          <em>Understand</em> the problem that has been identified
+          and <em>confirm</em> the intended users feel the <i>pain</i>
+          and <em>need</em> for a solution.
```

Comment:
Semantically, I think this should be an `<i>` instead of an `<em>` but it's not really worth spending that much time debating! :blush: